subsys/bluetooth/services/Kconfig.dis

```diff
@@ -0,0 +1,109 @@
+# Kconfig - Bluetooth GATT Device Information service
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig BT_GATT_DIS
+	bool "Enable BLE GATT Device Information service"
+	default n
+
+if (BT_GATT_DIS)
```

Comment:
the parenthesis are not needed (and inconsistent - I haven't seen them used like this elsewhere in the tree)

Reply:
Fixed